app/services/CompileService.scala

```diff
@@ -108,6 +72,43 @@ class CompileService @Inject()(
     }
   }
 
+  private def doCompile(currentVersionInfo: VersionInfo,
+                        newVersionInfo: VersionInfo,
+                        pstr: String,
```

Comment:
Can we put back the doCompile function back where it was. I can't see the changes because it was swapped down the file.